packages/typescriptlang-org/src/templates/play.tsx

```diff
@@ -111,7 +111,30 @@ const Play: React.FC<Props> = (props) => {
           div.style.webkitAnimation = ""
         })
 
-        document.getElementById("loading-message")!.innerHTML = `This version of TypeScript <em>(${tsVersion?.replace(/</g, "-")})</em><br/>has not been prepared for the Playground<br/><br/>Try <a href='/play?ts=${latestRelease}${document.location.hash}'>${latestRelease}</a> or <a href="/play?ts=next${document.location.hash}">Nightly</a>`
+        const loadingMessage = document.getElementById("loading-message")!
+        loadingMessage.textContent = ""
+
+        const em = document.createElement("em")
+        em.textContent = `(${tsVersion?.replace(/</g, "-")})`
```

Comment:
I assume the replace is not needed anymore, now that the text doesn't need to be sanitized?